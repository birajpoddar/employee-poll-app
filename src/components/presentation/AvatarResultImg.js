const AvatarResultImg = ({ user }) => (
	<img
		src={user.avatar}
		alt={user.name}
		className="navbar-brand"
		style={{
			borderRadius: '50%',
			height: '40px',
			border: '3px solid var(--bs-gray-600)',
			zIndex: 1,
			boxShadow: '0px 2px 2px 0px #000000aa',
			marginBottom: '-20px',
		}}
	/>
);

export default AvatarResultImg;
