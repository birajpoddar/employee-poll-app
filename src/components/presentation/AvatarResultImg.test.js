import { render } from '@testing-library/react';
import AvatarResultImg from './AvatarResultImg';

describe('AvatarResultImg Tests', () => {
	it('Snapshot Test with props', () => {
		const view = render(
			<AvatarResultImg
				user={{
					name: 'Zenobia Oshikanlu',
					avatar:
						'https://images.unsplash.com/photo-1599566150163-29194dcaad36?w=400&h=400&fit=facearea&facepad=3',
				}}
			/>
		);
		expect(view).toMatchSnapshot();
	});

	it('Snapshot Test without props', () => {
		const view = render(<AvatarResultImg user={{}} />);
		expect(view).toMatchSnapshot();
	});
});
