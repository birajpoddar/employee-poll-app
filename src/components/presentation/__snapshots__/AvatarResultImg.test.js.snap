// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AvatarResultImg Tests Snapshot Test with props 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <img
        alt="Zenobia Oshikanlu"
        class="navbar-brand"
        src="https://images.unsplash.com/photo-1599566150163-29194dcaad36?w=400&h=400&fit=facearea&facepad=3"
        style="border-radius: 50%; height: 40px; z-index: 1; box-shadow: 0px 2px 2px 0px #000000aa; margin-bottom: -20px;"
      />
    </div>
  </body>,
  "container": <div>
    <img
      alt="Zenobia Oshikanlu"
      class="navbar-brand"
      src="https://images.unsplash.com/photo-1599566150163-29194dcaad36?w=400&h=400&fit=facearea&facepad=3"
      style="border-radius: 50%; height: 40px; z-index: 1; box-shadow: 0px 2px 2px 0px #000000aa; margin-bottom: -20px;"
    />
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`AvatarResultImg Tests Snapshot Test without props 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <img
        class="navbar-brand"
        style="border-radius: 50%; height: 40px; z-index: 1; box-shadow: 0px 2px 2px 0px #000000aa; margin-bottom: -20px;"
      />
    </div>
  </body>,
  "container": <div>
    <img
      class="navbar-brand"
      style="border-radius: 50%; height: 40px; z-index: 1; box-shadow: 0px 2px 2px 0px #000000aa; margin-bottom: -20px;"
    />
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
